package.json

```diff
@@ -19,7 +19,7 @@
     "oracledb": "^3.1.2"
   },
   "devDependencies": {
-    "@dvsa/mes-test-schema": "3.19.0",
+    "@dvsa/mes-test-schema": "3.21.1",
```

Comment:
if this goes in the release it shouldn't update to `3.21.*` should stay at `3.20.*`